src/Routes/MyRoutes.js

```diff
@@ -0,0 +1,17 @@
+import { Route, Routes } from "react-router-dom";
+import { Dashboard, Login, Signup, Timer } from "../Pages";
+import Mockman from "mockman-js";
+const MyRoutes = () => {
+  return (
+    <>
+      <Routes>
+        <Route path="/" element={<Timer />} />
+        <Route path="/dashboard" element={<Dashboard />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
+        <Route path="/mock" element={<Mockman />} />
```

Comment:
Why are you using mockman, i am asking this because we don't have APIs for pomodoro.

Reply:
Yes I got it.
I don't have to use this.